packages/client/components/TeamPrompt/WorkDrawer/LinearIntegrationPanel.tsx

```diff
@@ -0,0 +1,133 @@
+import graphql from 'babel-plugin-relay/macro'
+import {useState} from 'react'
+import {useFragment} from 'react-relay'
+import {LinearIntegrationPanel_meeting$key} from '../../../__generated__/LinearIntegrationPanel_meeting.graphql'
+import useAtmosphere from '../../../hooks/useAtmosphere'
+import useMutationProps from '../../../hooks/useMutationProps'
+import linearSVG from '../../../styles/theme/images/graphics/linear.svg'
+import LinearClientManager from '../../../utils/LinearClientManager'
+import SendClientSideEvent from '../../../utils/SendClientSideEvent'
+import LinearIntegrationResultsRoot from './LinearIntegrationResultsRoot'
+import LinearProjectFilterBar from './LinearProjectFilterBar' // Corrected import name
+
+interface Props {
+  meetingRef: LinearIntegrationPanel_meeting$key
+}
+
+const LinearIntegrationPanel = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment LinearIntegrationPanel_meeting on TeamPromptMeeting {
+        teamId
+        id
+        viewerMeetingMember {
+          teamMember {
+            teamId
+            integrations {
+              linear {
+                api {
+                  query {
+                    viewer {
+                      id
+                    }
+                  }
+                }
+                auth {
+                  isActive
+                }
+                cloudProvider {
+                  id
+                  clientId
+                  serverBaseUrl
+                }
+              }
+            }
+            # Spread the fragment required by the updated LinearProjectFilterBar
```

Comment:
-1 What does that mean? We require the fragment because we use the component, it's obvious and will be caught by relay if missing.